include/NovelRT/Maths/QuadTree.h

```diff
@@ -1,9 +1,11 @@
 #ifndef NOVELRT_MATHS_QUADTREE_H
 #define NOVELRT_MATHS_QUADTREE_H
 
-#ifndef NOVELRT_H
-#error Please do not include this directly. Use the centralised header (NovelRT.h) instead!
-#endif
```

Comment:
I won't comment for every Maths type, but seeing as we do it in `Graphics.Vulkan` and `Graphics` - do we want to refer to including `Maths.h` for all of the Maths types instead of just completely removing the guards?

Reply:
This is a leftover from when we were unsure how we wished to structure the namespace headers, and was from the brief period I made headers self-contained.

Reply:
To be fair, if we aren't using most of these types once we finish getting Vulkan wrapped, I think it makes sense to not bother for now. Maybe we open a ticket for the types that will be sticking around? I don't think they _have_ to be handled here.